### PR TITLE
Gigalixir

### PR DESCRIPTION
**Pull request by chriserin**
Get elixir releases working so that we can deploy into Gigalixir

### CODE REVIEW

File: .tool-versions [hunk 1]
```diff
@@ -1,3 +1,3 @@
-elixir 1.9.2-otp-22
-erlang 22.0.7
+elixir 1.10.2
```
Comment (dkarter) [inline .tool-versions:1]: We should still use the `1.10.2-otp-22` version if we are using Erlang 22

Comment (chriserin) [reply .tool-versions:1]: @dkarter that version isn't listed in `asdf list-all elixir`

Comment (chriserin) [reply .tool-versions:1]: Looks like erlang and elixir are meant to be installed separately.

Comment (dkarter) [reply .tool-versions:1]: Really? that's weird... I see it on my end:
image

**Comment by jwworth** [comment]
Closing, with freedom to reopen if we decide to look into this again.

**Pull request closed without merging**